src/Detector/Node/NodeDetector.cs

```diff
@@ -253,6 +265,17 @@ private IEnumerable<FrameworkInfo> DetectFrameworkInfos(DetectorContext context)
                 detectedFrameworkResult.Add(frameworkInfo);
             }
 
+            // remove base frameworks if derived framework exists
+            if (frameworksSet.Contains("Gatsby") || frameworksSet.Contains("Next.js"))
+            {
+                detectedFrameworkResult.RemoveAll(x => x.Framework == "Angular");
+                detectedFrameworkResult.RemoveAll(x => x.Framework == "React");
+            }
+            if (frameworksSet.Contains("VuePress") || frameworksSet.Contains("Nuxt.js"))
```

Comment:
nit: would you mind adding a newline before this `if` block?

Reply:
Updated